Feature/storybook (#40)
infinum/eightshift-frontend-libs: blocks/init/custom/group/docs/readme.md

# Group Block

> Block used to provide gropup functionality to your project. This block is a inner blocks (repeater) block that expects children blocks to be present. It is designed to group multiple children block in to one block for a simpler configuration and handling.

Reviewer: ```suggestion
Block used to provide group functionality to your project. This block is an inner block (repeater) block that expects children blocks to be present. It is designed to group multiple children blocks into one block for simpler configuration and handling.
```

## Dependencies

None

## Implementation
1. Copy/Paste block folder in your project
2. Rename `eightshift-boilerplate` namespace to your project
3. Add or remove features you are going to use.
4. Implement project specific styles.

## Note

> This block uses depenedencies from blocks folder. If you are going to use this block in your project be sure to have other block present.

Reviewer: ```suggestion
This block uses dependencies from the blocks folder. If you are going to use this block in your project, be sure to have other block present.
```
